source/docs/application-notes/tuner-evolution.rst

```diff
@@ -0,0 +1,97 @@
+Tuner and an evolution in configuration
+=======================================
+
+*Authored by Dalton*
+
+.. image:: images/tuner-evolution/tuner-v1.png
+   :alt: Image of Tuner v1
+
+Since the introduction of the CTRE Toolsuite (pre-2018), we at CTR have strived to provide intuitive means of configuring and utilizing our products. In 2018, we launched Phoenix Tuner (now lovingly referred to as Tuner v1). Tuner v1 introduced features like: batch firmware upgrading like devices, diagnostic server deployment, self tests, plotting and control.
+
+With Tuner, and by extension diagnostics, we have several primary objectives:
+
+- Ease of debugging (exposed via self test).
+- Seamless setup experience.
+- Support and integrate our extension feature-set.
+
+Tuner v1 was and is great, but we wanted to do more. In 2023, we introduced Tuner X.
```

Comment:
Maybe say "for the 2023 season" or "Fall 2022"?
Just '2023' makes it sound like we just released it when it's actually been around for almost a year at this point.